aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -211,77 +211,95 @@ SparseTensor& add_out_sparse_cpu(SparseTensor& r, const SparseTensor& t, const S
   Tensor t_values = t._values();
   LongTensor src_indices = src._indices();
   Tensor s_values = src._values();
-  LongTensor r_indices = at::empty({sparse_dim, max_nnz}, t_indices.options());
-  Tensor r_values = new_values_with_size_of(s_values, max_nnz).zero_();
   r.resize_as_(src);
-  get_sparse_impl(r)->set_indices_and_values_unsafe(r_indices, r_values);
 
-  int64_t blockSize = r_values.stride(0);
-  int64_t cmp, d;
-  int64_t r_i = 0, t_i = 0, s_i = 0;
+  if (s_values.is_contiguous() && t_values.is_contiguous()) {
```

Comment:
There is no change in this if-branch compared to the original code - I only indented it.